tests/end2end/test_storage_volume.py

```diff
@@ -203,6 +203,9 @@ def test_stovol_crud(dpm_mode_cpcs):  # noqa: F811
             stovol = stogrp.storage_volumes.create(stovol_input_props)
 
             for pn, exp_value in stovol_input_props.items():
+                assert pn in stovol.properties, (
+                    f"Input property {pn!r} is not in created storage volume:\n"
+                    f"{stovol!r}")
```

Comment:
consistency: you are using the new f"{}" style here, but keep the old style in other places? 
Maybe run that "update" script on the files you modify?

Reply:
For reasons I do not understand yet, the pyupgrade tool I used for the upgrade of the code to Python 3.8 converted only a subset of the format strings to f-strings. I would like to convert the remaining ones in a separate PR.